resource-manager/src/main/java/org/ihtsdo/otf/resourcemanager/ResourceManager.java

```diff
@@ -291,7 +348,7 @@ public void moveResource(final String fromResourcePath,
 							 final String toResourcePath) throws IOException {
 		try {
 			if (resourceConfiguration.isUseCloud()) {
-				s3MoveResource(fromResourcePath, toResourcePath);
+				s3MoveResource(fromResourcePath, toResourcePath, true);
```

Comment:
@pgwilliams The method would copy then delete. I added an additional parameter to control whether delete was optional.